### PR TITLE
[MIRROR] Fixes some fluff weirdness with Virgo-ported prosthetics

### PR DESCRIPTION
**Pull request by VirgoBot**
Original PR: https://github.com/PolarisSS13/Polaris/pull/7907
--------------------
Zeng-Hu Tajaran Heads weren't tagged as being for Taj. Is now + acts as default option when taj pick Zeng-Hu.
Eggnerd Unathi-prosthetics were available to Everybody. Made Unathi-only. (Also, refluffed the name)
Talon LLC doesn't exist in our lore but was made a 'default' option on prosfabs. Changed to a NanoTrasen own-brand ultra-cheapo 'lifelike' model named Metro(polis, geddit).

### CODE REVIEW

**Comment by Verkister** [comment]
This is eggnerd erasure smh

**Comment by Verkister** [comment]
They really going around and editing personal fluff __vr files_ without even using another file huh

**Comment by Novacat** [comment]
This PR will have to be put on hold until after Kaleidoscope is merged anyway.

File: code/modules/organs/robolimbs_vr.dm [hunk 1]
```diff
@@ -53,13 +53,17 @@ var/const/cyberbeast_monitor_styles = "blank=cyber_blank;\
 
 // verkister : Rahwoof Boop
 /datum/robolimb/eggnerdltd
-	company = "Eggnerd Prototyping Ltd."
-	desc = "This limb has a slight salvaged handicraft vibe to it. The CE-marking on it is definitely not the standardized one, it looks more like a hand-written sharpie monogram."
+	company = "Uesseka Prototyping Ltd."
+	desc = "This limb seems meticulously hand-crafted, and distinctly Unathi in design."
```
Comment (KasparoVy) [inline code/modules/organs/robolimbs_vr.dm:57]: I like the re-fluff.

**Comment by Verkister** [comment]
This PR is literally just meddling with _our fluff item _vr file_, _applies unnecessary species restrictions_ to our robolimbs, and also the _"Eggnerd" set was literally **my** personal fluff item_ that I made accessible for public i char setup and ingame orders. **This PR would be a straight up harm merge.** :v

**Pull request merged**